pagopa/pn-frontend
# PN-10587-fe-api-institutions-and-products-pa

## Short Description
Migrate the old Institutions and Products API to the BFF's API.

## List of Changes Proposed in This Pull Request
- Generated models from the BFF OpenAPI.
- Updated `getInstitutions()` and `getInstitutionsProducts()` with the BFF's API.
- Updated related tests.
- Updated related mocks.

## How to Test
Start PA and verify if you can see the list of institutions and products on the dashboard.
